lib/models.js

```diff
@@ -985,6 +1005,10 @@ class TypeConstraint extends Constraint {
       "onValue": this.onValue
     }, super.toJSON())
   }
+
+  toString() {
+    return `TypeConstraint (${this.isA.fqn})`
```

Comment:
If there is a `path`, please represent it in `toString`.  Similarly, if `onValue` is set, please indicate that too.